components/core/tools/scripts/utils/build.py

```
@@ -0,0 +1,117 @@
+# Set up root logger
+root_logger = logging.getLogger()
+root_logger.setLevel(logging.INFO)
+root_logger.addHandler(logging_console_handler)
+
+# Create logger
+logger = logging.getLogger(__name__)
+
+# Number of parent directories to reach project root from this script
+PROJECT_ROOT_DEPTH = 6
+
+def _run_subprocess(cmd: List[str], cwd: Optional[Path] = None) -> None:
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)